server/validation/validation.ts

```diff
@@ -0,0 +1,37 @@
+import Validator, { ErrorMessages, Rules } from 'validatorjs'
+import isISO8601 from 'validator/lib/isISO8601'
+
+export interface Error {
+  href: string
+  text: string
+}
+Validator.register(
+  'datetime',
+  value => {
+    if (typeof value === 'string') {
+      return isISO8601(value)
+    }
+    return false
+  },
+  'Enter a real date time, like 2020-03-23T12:00:00 or 2020-03-23'
+)
```

Comment:
this is used by developers so it is probably acceptable to have a IUSO data format